docs/content/direct/core-chart.md

```diff
@@ -1,5 +1,17 @@
 # KubeStellar Core chart usage
 
+## Table of Contents
+- [Overview](#overview)
+- [Pre-requisites](#pre-requisites)
+- [KubeStellar Core Chart values](#kubestellar-core-chart-values)
+- [KubeStellar Core Chart usage](#kubestellar-core-chart-usage)

```

Comment:
There are two headings with exactly the same words in them ("KubeStellar Core chart usage"). The link given here goes to the first, but you want to go to the second instead. Better yet, reword one or both so that this confusing situation does not exist.